tests/forms_tests/widget_tests/test_multiwidget.py

```diff
@@ -118,6 +123,42 @@ def test_constructor_attrs(self):
             '<input id="bar_1" type="text" class="small" value="lennon" name="name_1" />'
         ))
 
+    def test_constructor_attrs_with_type(self):
+        attrs = {'type': 'number'}
+        widget = MyMultiWidget(widgets=(
+            TextInput,
+            TextInput,
+            TextInput(),
+        ), attrs=attrs)
+        self.check_html(widget, 'code', ['1', '2', '3'], html=(
+            '<input type="number" value="1" name="code_0" />'
```

Comment:
On 1.10, the output is: `'<input type="number" value="1" name="code_0" />'
'<input type="number" value="2" name="code_1" />'
'<input type="number" value="3"  name="code_2" />'`.